packages/react/src/transactions/adapters/__tests__/CollectPublicationCallGateway.spec.ts

```diff
@@ -81,4 +133,107 @@ describe(`Given an instance of the ${CollectPublicationCallGateway.name}`, () =>
       expect(unsignedCall.nonce).toEqual(nonce);
     });
   });
+
+  describe(`when calling the "${CollectPublicationCallGateway.prototype.createProxyTransaction.name}" method`, () => {
+    describe('when relaying a FreeCollectRequest', () => {
```

Comment:
These tests read odds: "when ... when" in a behaviour focused tests style, also the extra "and" makes it difficult to distinguish what is the event (the when) and what are the preconditions (the givens):

I think the `when relaying a FreeCollectRequest` already implies calling the `createProxyTransaction`, it's just expressed in more domain language. I don't have a preference between one or the other as far as now that we merged the 2 gateways together we keep the wording consistent with the other method tests.